apps/src/netsim/NetSimVizSimulationNode.js

```diff
@@ -56,6 +62,7 @@ NetSimVizSimulationNode.inherits(NetSimVizNode);
  */
 NetSimVizSimulationNode.prototype.configureFrom = function (sourceNode) {
   this.correspondingNodeID_ = sourceNode.entityID;
+  this.correspondingNodeUuid_ = sourceNode.uuid;
```

Comment:
Sure.  I think I got this habit from the old [MSDN rules for capitalization of acronyms](https://msdn.microsoft.com/en-us/library/vstudio/ms229043(v=vs.100).aspx#CapForAcronyms) wherein two-character acronyms could be all capitalized but 3+ character acronyms only capitalize the leading letter.

Re-reading those rules, ID is an abbreviation, not an acronym, and should be written `id` or `Id` unlike two-letter acronyms.  I'll go with `Id` / `Uuid`.  Maybe I can do a pass to normalize this throughout NetSim soon.